modin/experimental/xgboost/xgboost.py

```diff
@@ -60,20 +75,20 @@ class Booster(xgb.Booster):
     """
     A Modin Booster of XGBoost.
 
-    Booster is the model of xgboost, that contains low level routines for
+    Booster is the model of XGBoost, that contains low level routines for
     training, prediction and evaluation.
 
     Parameters
     ----------
-    params : dict. Default is None
+    params : dict, optional
         Parameters for boosters.
-    cache : list
+    cache : list, default: empty
         List of cache items.
-    model_file : string/os.PathLike/Booster/bytearray
-        Path to the model file if it's string or PathLike.
+    model_file : string/os.PathLike/xgb.Booster/bytearray, optional
+        Path to the model file if it's string or PathLike or xgb.Booster.
     """
 
-    def __init__(self, params=None, cache=(), model_file=None):
+    def __init__(self, params=None, cache=(), model_file=None):  # noqa
```

Comment:
can we disable only relevant checks instead of disabling everything?

Reply:
Disabled only `MD01` now. Thanks.